common/app/views/fragments/page/email/body.scala.html

```diff
@@ -13,7 +13,7 @@
                                         <tr>
                                             <td class="no-pad">
                                                 @content
-                                                @fragments.email.support(page)
+                                                <!-- Support Ask Placeholder -->
```

Comment:
Any chance you can name this as 'Support Ask Placeholder - Footer' , as we are likely to have multiple of these in various locations in the future.

Reply:
Sure, not a problem. I will do that now.